test/conftest.py

```diff
@@ -51,6 +51,7 @@ def host_export_root_dir() -> Path:
 RESOURCES_DIR = TEST_DIR / "resources"
 RESOURCES_OMOP_DIR = RESOURCES_DIR / "omop"
 RESOURCES_OMOP_DICOMWEB_DIR = RESOURCES_DIR / "omop-dicomweb"
+SECONDS_TO_WAIT_FOR_EXPORT = 301
```

Comment:
this has increased from 211 because we're now doing an additional download and upload of the data (downloading raw data from anon, anonymising, then re-uploading and waiting for the study to stabilise), which slows things down